discojs/src/aggregation.ts

```diff
@@ -22,7 +22,7 @@ function applyArithmeticToWeights (peersWeights: List<Weights>, tfjsArithmeticFu
 }
 
 export function sumWeights (peersWeights: List<Weights>): Weights {
-  return applyArithmeticToWeights(peersWeights, tf.add)
+  return applyArithmeticToWeights(peersWeights.toList(), tf.add)
 }
 
 export function subtractWeights (peersWeights: List<Weights>): Weights {
```

Comment:
same here as well for subtraction? (though ok that method seems unused anyway and not clear)

Reply:
good catch, it was dangling from a previous iteration; there is not need to convert a `List` to a `List` (: